src/main/io/ledstrip.c

```diff
@@ -938,6 +938,30 @@ static void applyLarsonScannerLayer(bool updateNow, timeUs_t *timer)
     }
 }
 
+static void applyRainbowLayer(bool updateNow, timeUs_t *timer)
+{
+    int delta = 24;
+    static int offset = 0;
+
+    if (updateNow) {
+        *timer += HZ_TO_US(60);
```

Comment:
Literal should also check extracted to a named constant

Reply:
Is this for the `60` in HZ_TO_US(60)? If so would you like me to update all occurrences of HZ_TO_US(60) in this file?

Reply:
I would do something like:
```cpp
#define TIMER_UPDATE_PERIOD HZ_TO_US(60)
```

But if it occurs more than once inside this file, maybe better to leave it aside for another PR (with refactoring).

Reply:
@DieHertz : IMO different effects have different update frequencies. It may be easier to see update frequency close to animation code ...

Reply:
Then define the macro constant right next to the code and the undefine it.
Or at least use `const` specifier, as we do in C++, it should lead to no runtime footprint for the constant.